modules/ROOT/partials/configuration/menu.adoc

```diff
@@ -1,11 +1,11 @@
 [[menu]]
 == `+menu+`
 
-This option allows you to specify which menus should appear on {productname}'s menu bar and the xref:available-menu-items.adoc[items] that should appear within the menus themselves.
+This option allows you to define what menus are available to the xref:menus-configuration-options.adoc#menubar[menubar option] and the xref:available-menu-items.adoc[menu items] that should appear within the menus themselves.
 
-To specify the menus that should appear on {productname}'s menu bar, the menu option should be provided with a JavaScript object containing a property for each menu. These properties should contain a JavaScript object themselves with properties `+title+` and `+items+`.
+To define the menus available in {productname}, the `+menu+` option should be provided with a JavaScript object containing a property for each menu. These properties should contain a JavaScript object themselves with properties `+title+` and `+items+`.
 
-The `+title+` property should contain a string with the name of the `+menu+`. The `+items+` field should contain a space separated list of the controls that should populate the `+menu+`.
+The `+title+` property should contain a string with the display name of the `+menu+`. The `+items+` field should contain a space separated list of the controls that should populate the `+menu+`.
```

Comment:
Not sure if you want to remove this section or maybe add more details to the above suggestion?